time/chrono.h

```diff
@@ -28,20 +28,52 @@ SOFTWARE.
 #define BRICKS_TIME_CHRONO_H
 
 #include <algorithm>
+#include <thread>
 #include <chrono>
 
 #include "../port.h"
 #include "../strings/fixed_size_serializer.h"
 
 namespace bricks {
-
 namespace time {
 
 enum class EPOCH_MILLISECONDS : uint64_t {};
 enum class MILLISECONDS_INTERVAL : uint64_t {};
 
+}  // namespace bricks::time
+}  // namespace bricks
+
+inline bricks::time::MILLISECONDS_INTERVAL operator-(bricks::time::EPOCH_MILLISECONDS lhs,
```

Comment:
Because I want to strictly separate `unit64_t`-s from `EPOCH_MILLISECOND`-s.

(Also, this pull request just rearranges sections, this is not new code.)